packages/functions/src/coordinator/CoordinatorBookAppointmentHandler.ts

```diff
@@ -1,13 +1,31 @@
-import { FunctionsApi } from "@zm-blood-components/common";
+import {
+  FunctionsApi,
+  CoordinatorRole,
+  MANUAL_DONOR_ID,
+} from "@zm-blood-components/common";
 import { bookAppointment } from "../common/BookAppointmentHelper";
+import * as CoordinatorDAL from "../dal/AdminDataAccessLayer";
 
 export default async function (
   request: FunctionsApi.CoordinatorBookAppointmentRequest,
   callerId: string
 ): Promise<FunctionsApi.BookAppointmentResponse> {
+  if (request.donorId !== MANUAL_DONOR_ID) {
```

Comment:
I think we should throw here.

If (and when) we want to support this feature, let's do it in a different PR

Reply:
why in a different PR? all the functionality is already written here..

Reply:
I don't think this is the right way to do it,
But more important, this PR is very big and has been open for over 14d.
We need to close it.

Manually marking a manual donor or a real donor are two different features, and they deserve different PRs

Reply:
but we already been throw all the developing process here. why to erase that work?

Reply:
and originally that was the hole purpose of this PR - to add the option of a manual donor

Reply:
The requirement is to add a manual donor, a dummy one. Not an existing real user.

Assigning a real user requires more things, like notifying the users and more.

If you want to keep the work that was done here, move it to a branch so we could use it later

Reply:
its not a real one. this line is sais : "if this is a manual donor dont check if the coordinator has permissions to this donor, and return bookAppointment(...)"

Reply:
the manual donors details stays in the appointment details, as planed..

Reply:
Even the point that we have some disagreements here point to the face this needs to be in a different PR.

Let me ask you this way - if you remove this code and throw here, would this PR be useless or have some meaning?

if it has meaning - lets merge it without this and work on this logic in a different PR.